packages/core-react/src/components/Button/index.js

```diff
@@ -9,14 +9,16 @@ export default function Button({
   compact,
   danger,
   disabled,
+  className,
   ...props
 }) {
   return (
     <Tag
       {...props}
       className={clsx('ray-button', `ray-button--${type}`, {
         'ray-button--compact': compact,
-        'ray-button--danger': danger
+        'ray-button--danger': danger,
+        [`${className}`]: !!className
```

Comment:
- do you need to interpolate?
- do you need to type coerce? 

```suggestion
        [className]: className
```

Reply:
hmm, good point